src/containers/MusicDashboard/index.tsx

```diff
@@ -32,12 +32,12 @@ type MusicDashboardState = {
   more: string;
   [x: number]: number;
   songState: ISong;
-  navState:{navSong:boolean, navPhoto:boolean, navTour: boolean};
+  navState:{ navSong:boolean, navPhoto:boolean, navTour: boolean };
 };
 
 const InitialState = {
   songState: {
-    image: '', composer: '', year: 2020, album: '', title: '', url: '', artist: '', category: 'original', _id: '',
+    image: '', composer: '', year: 2021, album: '', title: '', url: '', artist: '', category: 'original', _id: '',
```

Comment:
year should be determined by current year using the Date object here, not hard coded